src/Cake.Core/Scripting/ScriptRunner.cs

```diff
@@ -48,6 +52,10 @@ public sealed class ScriptRunner : IScriptRunner
             {
                 throw new ArgumentNullException("log");
             }
+            if (configuration == null)
```

Comment:
@patriksvensson I could be wrong about this StyleCop rule. I am seeing it in a few places now :smile: